docker-compose.yml

```diff
@@ -76,12 +76,20 @@ services:
       CLICKHOUSE_DB: default
       CLICKHOUSE_USER: clickhouse
       CLICKHOUSE_PASSWORD: clickhouse
+      # since clickhouse is memory intensive, use these two variables to tune your memory usage
+      CLICKHOUSE_MAX_MEMORY_USAGE: "800000000"  # ~800MB
+      CLICKHOUSE_MAX_MEMORY_USAGE_FOR_USER: "800000000"  # ~800MB
```

Comment:
@electricjimi Where did you derive those defaults from? Are those public values or are you setting something that is deliberately smaller than common standards?

If we'd go down this route, we should use the official defaults as defaults and allow an overwrite.